canarieapi/api.py

```diff
@@ -357,25 +363,24 @@ def stats(route_name: str, api_type: APIType) -> ResponseReturnValue:
         )
         return error_html, 503
 
-    # Gather route stats
+    monitor_info = []
     cron_info = collect_cron_access_stats(route_name, database=db)
 
-    monitor_info = [
+    if APP.config.get("PARSE_LOGS"):
+        service_stats.append(("invocations", cron_info["invocations"]))
+        monitor_info.append(("lastInvocationsUpdate", cron_info["last_log_update"]))
+
+    monitor_info.extend([
         ("lastAccess", cron_info["last_access"]),
```

Comment:
Maybe consider having `"Disabled"` instead of the default `"Never"` so that it is more obvious that this status is expected when `PARSE_LOGS = False` was specified?

Reply:
I think I was not clear what "lastAccess" referred to in this case. If it refers to the last time this route was accessed by a user then it should be displayed conditionally if monitoring is enabled. I've moved it up into the `if` block (above) to reflect this

Reply:
It refers more to when the logs were last accessed (aka checked). So, more like a "last updated". 
I don't think the last access time of the endpoint is relevant for anyone. It is usually used to validate if the logs were parsed recently or are ~1min old (or more depending on crontab config). The `"Never"` is usually an indication that something is misconfigured, or that the instance encountered some kind of error. 

I want to avoid an *expected* `"Never"` (because log parsing is disabled) to be misinterpreted as something going wrong in the service. Alternatively, you could output `parseLogsEnabled: true|false` directly in the response. That would be even more explicit than the `"Disabled"` previously suggested.

Reply:
This still remains to be addressed.